packages/SwingSet/src/types-internal.js

```diff
@@ -0,0 +1,15 @@
+import './types-exported';
+
+export {};
+
+/**
+ * @typedef { string } VatID
```

Comment:
Nit: avoid redefining if there is already a definition somewhere else (except if you mean to have a different but "compatible" definition for external use, such as a subset)

```suggestion
 * @typedef { import('./types-exported.js'}.VatID } VatID
```

Reply:
oh in fact I should just have `types-internal.js` import everything, with `import './types-exported.js'`.. the internal types are a superset of the exported ones

Also, `VatID` wasn't supposed to be visible to the outside world, at least not as part of the current public API. Vats don't learn their own IDs.